lib/mix/lib/mix/project.ex

```diff
@@ -234,7 +234,7 @@ defmodule Mix.Project do
   A `post_config` can be passed that will be merged into
   the project configuration.
 
-  `fun` is called with the `Mixfile` of the given project as
+  `fun` is called with the `mixfile` of the given project as
```

Comment:
What do you think about: `fun` is called with the module name of the given `Mix.Project`